binaries/cli/src/lib.rs

```diff
@@ -537,6 +551,61 @@ fn run(args: Args) -> eyre::Result<()> {
             .context("failed to run dora-daemon")?
         }
         Command::Runtime => dora_runtime::main().context("Failed to run dora-runtime")?,
+        Command::Self_ { command } => match command {
+            SelfSubCommand::Update { check_only } => {
+                println!("Checking for updates...");
+
+                #[cfg(target_os = "linux")]
+                let bin_path_in_archive = format!("dora-cli-{}/dora", env!("TARGET"));
```

Comment:
```suggestion
                let bin_path_in_archive = format!("dora-cli-{}/dora", env!("CARGO_BUILD_TARGET"));
```

I think this should remove the need for the build script

Reply:
I don't think CARGO_BUILD_TARGET is actually an env variable? I mean, it's not present in my code x)

Reply:
Ah ok indeed!

Sorry!